ci/Dockerfile

```diff
@@ -30,9 +30,12 @@ ENV PATH /root/.cargo/bin/:$PATH
 RUN rustup component add rustfmt llvm-tools-preview clippy
 
 # install build tools
-RUN cargo install cargo-llvm-cov cargo-nextest cargo-udeps cargo-hakari cargo-sort cargo-make cargo-cache \
+RUN cargo install cargo-llvm-cov cargo-nextest cargo-udeps cargo-hakari cargo-sort cargo-make cargo-cache sccache \
     && cargo install --git https://github.com/risinglightdb/sqllogictest-rs --rev dbadddb --bin sqllogictest --locked \
     && cargo cache -a \
     && rm -rf "/root/.cargo/registry/index" \
     && rm -rf "/root/.cargo/registry/cache" \
     && rm -rf "/root/.cargo/git/db"
+
+ENV RUSTC_WRAPPER=sccache
```

Comment:
Hard to believe it's so easy to use. 🥵

Reply:
🥵 The hard part for me is to learn to how to configure AWS

Reply:
Built by Mozilla after all 😍